src/compute-types/src/dyncfgs.rs

```diff
@@ -99,6 +99,13 @@ pub const LGALLOC_SLOW_CLEAR_BYTES: Config<usize> = Config::new(
     "Clear byte size per size class for every invocation",
 );
 
+/// The denominator `n` in the growth rate `1 + 1/(n-1)` for `ConsolidatingVec` growth.
+pub const CONSOLIDATING_VEC_GROWTH_DENOMINATOR: Config<usize> = Config::new(
+    "consolidating_vec_growth_denominator",
+    2,
```

Comment:
I'd be okay with defaulting this to our target (`11`?) and then opting out were we observe issues. Seems safe enough, considering that this trades of less memory for more CPU and we usually don't have CPU issues.

Reply:
What do you think about starting this way, then trying out some new values, then a future item that cuts us over to something like 11 as a default? I'm flexible, but if you think that starting at 11 is the right way to make sure we actually test this and do it I'm happy to make the change!

Reply:
No strong opinion from me either! If you want to go more slowly that's fine with me.

Reply:
I think this is a bigger question now that it seems we aren't tracking the standard Rust behavior when we increase the capacity. We could try and sort that out to make it look like doubling (e.g. by ensuring that the capacity grows by `1 + 1/(damp + 1)`, by doing the math to hit this target based on the current length).

Reply:
Most recent commit tries to handle this.